tests/pre_merge/utils/loggers/test_get_logger.py

```diff
@@ -9,8 +9,6 @@
 from omegaconf import OmegaConf
 
 try:
```

Comment:
Why is this removed? If we don't try importing wandb then this try statement will always pass

Reply:
it's an autofix by `ruff`. It complains that the import is not used.

Reply:
Ignoring that line now